apps/sage_intacct/views.py

```
@@ -244,3 +247,91 @@ def post(self, request: Request, *args, **kwargs) -> Response:
+            )
+
+    def decode_refresh_token(self, refresh_token: str, workspace_id: int) -> str:
+        """
+        Decode refresh token
+        :param refresh_token: Refresh token
+        :param workspace_id: Workspace ID
+        :return: Company ID
+        """
+        try:
+            return jwt.decode(refresh_token, options={"verify_signature": False})['cnyId']
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)